Code/GraphMol/Substruct/catch_tests.cpp

```diff
@@ -843,15 +843,69 @@ M  END)CTAB"_ctab;
 }
 
 TEST_CASE("Github #7295", "CIS/TRANS in aromatic ring") {
-    SECTION("Smart CIS/TRANS bonds should match single or aromatic"){
-      SubstructMatchParameters ps;
+  SECTION("Smart CIS/TRANS bonds should match single or aromatic") {
+    SubstructMatchParameters ps;
 
-      auto query = "[O:1]=[c:2]1/[c:3](=[C:4]/[c:5]2:[c:10]:[c:9]:[c:8]:[c:7]:[c:6]:2):[s:11]:[c:12]2:[n:13]:1-[N:14]-[C:15]-[N:20]-[N:21]=2"_smarts;
-      auto mol = "[O:1]=[c:2]1/[c:3](=[CH:4]/[c:5]2[cH:6][cH:7][cH:8][cH:9][cH:10]2)[s:11][c:12]2[n:13]1[NH:14][C:15]1([CH2:16][CH2:17][CH2:18][CH2:19]1)[NH:20][N:21]=2"_smiles;
-      CHECK(SubstructMatch(*mol, *query, ps).size() == 1);
-      ps.useChirality = true;
-      CHECK(SubstructMatch(*mol, *query, ps).size() == 1);
-      auto mol2 = "[O:1]=[c:2]1\\[c:3](=[CH:4]/[c:5]2[cH:6][cH:7][cH:8][cH:9][cH:10]2)[s:11][c:12]2[n:13]1[NH:14][C:15]1([CH2:16][CH2:17][CH2:18][CH2:19]1)[NH:20][N:21]=2"_smiles;
-      CHECK(SubstructMatch(*mol2, *query, ps).size() == 0);
-    }
+    auto query =
+        "[O:1]=[c:2]1/[c:3](=[C:4]/[c:5]2:[c:10]:[c:9]:[c:8]:[c:7]:[c:6]:2):[s:11]:[c:12]2:[n:13]:1-[N:14]-[C:15]-[N:20]-[N:21]=2"_smarts;
+    auto mol =
+        "[O:1]=[c:2]1/[c:3](=[CH:4]/[c:5]2[cH:6][cH:7][cH:8][cH:9][cH:10]2)[s:11][c:12]2[n:13]1[NH:14][C:15]1([CH2:16][CH2:17][CH2:18][CH2:19]1)[NH:20][N:21]=2"_smiles;
+    CHECK(SubstructMatch(*mol, *query, ps).size() == 1);
+    ps.useChirality = true;
+    CHECK(SubstructMatch(*mol, *query, ps).size() == 1);
+    auto mol2 =
+        "[O:1]=[c:2]1\\[c:3](=[CH:4]/[c:5]2[cH:6][cH:7][cH:8][cH:9][cH:10]2)[s:11][c:12]2[n:13]1[NH:14][C:15]1([CH2:16][CH2:17][CH2:18][CH2:19]1)[NH:20][N:21]=2"_smiles;
+    CHECK(SubstructMatch(*mol2, *query, ps).size() == 0);
+  }
 }
+
+TEST_CASE(
```

Comment:
Looks good, should probably check that a target with a query in a bond also doesn't match, but that's low priority I think.